src/models/accountrole.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
```

Comment:
Give names for model files in camel case

Reply:
yes